test-suite/src/array.rs

```diff
@@ -0,0 +1,43 @@
+use {
+    crate::*,
+    gluesql_core::prelude::{Payload, Value::*},
+};
+
+test_case!(array, {
+    let g = get_tester!();
+    g.run("CREATE TABLE Test (id INTEGER DEFAULT 1,surname TEXT NULL, name LIST NOT NULL);")
```

Comment:
for `array` test case, 'surname' field use looks unnecessary which is not related to `LIST` data type.
and..
could we have also simple `SELECT` query based test cases?
e.g.
```sql
SELECT ['hello', 1, 2, True];

SELECT ['a', 'b', 'c'][1];
```
thanks to array expression, now we can provide the above cases support.

Reply:
I have applied this changes